docs/source/guide/storage.md

```diff
@@ -295,11 +304,11 @@ In the Label Studio UI, do the following to set up the connection:
 2. For a specific project, open **Settings > Cloud Storage**.
 3. Click **Add Source Storage**.  
 4. In the dialog box that appears, select **Local Files** as the storage type.
-5. Specify the name of the local directory.
+5. Specify the name of the local directory. Label Studio assumes a root directory of `\`. To specify a different directory, see [Run Label Studio on Docker and use local storage](start.html#Run_Label_Studio_on_Docker_and_use_local_storage).
```

Comment:
again `/` , and maybe mentioning here that "local directory" is relative to the root directory? Or does it already stem from it?

Reply:
I think it's worth making that clearer, will update thanks